data/pages/tr/release

```diff
@@ -0,0 +1,27 @@
+<big>CachyOS 24.02</big>
+
+Bu 2024'teki ikinci dağıtımımız ve esasen çekirdek güncellemelerini, büyük glibc güncellemesini ve arındırılmış gerçek zamanlı imaj ortamını (Live ISO) içeren bir bakım sürümüdür. Bu sürüm esasen Plasma 6 yayınlanmadan ve Qt6'ya getirdiği değişikliklerden önce oluşturulmuş bir imaj dosyası bulunması maksadıyla yayınlandı.
```

Comment:
Instead of "Bu 2024'teki ikinci dağıtımımız" I think "Bu 2024'de çıkardığımız ikinci sürümümüz" is a lot more suitable.